pkg/standalone/stop_windows.go

```diff
@@ -29,7 +29,7 @@ func Stop(appID string) error {
 
 	for _, a := range apps {
 		if a.AppID == appID {
-			eventName, _ := syscall.UTF16FromString(fmt.Sprintf("dapr_cli_%v", a.PID))
+			eventName, _ := syscall.UTF16FromString(fmt.Sprintf("dapr_cli_%v", a.CliPID))
```

Comment:
@shubham1172 Why aren't  we checking for independent daprd process here ?

Reply:
Instead of using `kill`, as for Linux/Darwin, this employs a named events approach. This is the event handler that shuts down the CLI and subsequently the Daprd process:
https://github.com/dapr/cli/blob/6d3026b0cc237b6fc74353ed883846594dca22ab/cmd/shutdown_windows.go#L33-L41

Unfortunately, this means that independent daprd processes cannot be stopped. Let me create another issue to potentially track that in future, but it's out of scope for this PR. This change above does not affect anything as `PID` is now `CliPID`.

Reply:
Opened a new issue: https://github.com/dapr/cli/issues/1034